internal/core/items/image.rs

```diff
@@ -108,6 +108,15 @@ impl Item for ImageItem {
         (*backend).draw_image(self, self_rc, size, &self.cached_rendering_data);
         RenderingResult::ContinueRenderingChildren
     }
+
+    fn bounding_rect_for_geometry(
+        self: core::pin::Pin<&Self>,
+        _window_adapter: &Rc<dyn WindowAdapter>,
+        _self_rc: &ItemRc,
+        geometry: LogicalRect,
+    ) -> LogicalRect {
+        geometry
```

Comment:
The items that don't draw anything could return an empty rectangle.
That way we don't need to mark the area as dirty if a TouchArea moves

Reply:
Good idea :).